app.js

```diff
@@ -167,9 +190,21 @@ const fmt = {
 	},
 
 	background_color: function(event) {
-		const date_end = new Date(event.date.end || event.date.start)
-		return date_end <= new Date() ? 'opacity-50 hover:opacity-100 transition-opacity duration-200 ease-in-out' : ''
+		const date_end = new Date(event.date.end || event.date.start).getTime() + 86400000
+		return date_end <= new Date().getTime() ? 'opacity-50 hover:opacity-100 transition-opacity duration-200 ease-in-out' : ''
 	},
+
+	is_active: function (event) {
+		if (event.cancelled) {
+			return false
+		}
+
+		if (event.date.end) {
+			return new Date(event.date.start).getTime() <= new Date().getTime() && new Date().getTime() < new Date(event.date.end).getTime() + 86400000
```

Comment:
<img width="403" alt="CleanShot 2021-01-01 at 15 23 51@2x" src="https://user-images.githubusercontent.com/1835434/103440398-5c4df780-4c45-11eb-970a-6c5fb2836fb2.png">

why do we add getTime() everywhere? comparing dates works without it. (see multiple places in the file)

Reply:
Zajo zas neprepol jazyk :joy: 

lebo som nenašiel rozumnejší spôsob ako docieliť toto: `< new Date(event.date.end).getTime() + 86400000`

Reply:
moze byt. tiez nepoznam/nenasiel som lepsie.
ale potom by som to daval len na tie miesta kde naozaj nieco pripocitavas